.bingo/mockery.mod

```diff
@@ -1,7 +1,5 @@
 module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
 
-go 1.24
+go 1.24.2
```

Comment:
Would you mind re-generating this file using:

```
rm .bingo/mockery.mod
GOTOOLCHAIN=go1.24.0 ~/go/bin/bingo-v0.9.0 get -v github.com/vektra/mockery/v3@v3.4.0
```

If you get and error like `Error: parse go version: unexpected go version output; expected 'go version go<semver> ...; found go: downloading go1.24.0 (linux/amd64)` just run the command a second time.

This would allow us to still allow building with older go releases. (like some in some of our ci pipelines do)

Reply:
Yeah, happy to. I don't tend to use bingo.

Reply:
Because it had no differences other than replacing `go 1.24.2` with `go 1.24.0`, should I manually adjust it back to just `go 1.24`? or leave it with `go 1.24.0`?

Reply:
go 1.24.0 is fine

Reply:
Pushed :+1: